kakarot_scripts/constants.py

```diff
@@ -95,14 +95,7 @@
             f"STARKNET_NETWORK {os.environ['STARKNET_NETWORK']} given in env variable unknown"
         )
 else:
-    NETWORK = {
-        "name": os.getenv("RPC_NAME", "custom-rpc"),
-        "rpc_url": os.getenv("RPC_URL"),
-        "explorer_url": "",
-        "devnet": False,
-        "check_interval": float(os.getenv("CHECK_INTERVAL", 0.1)),
-        "max_wait": float(os.getenv("MAX_WAIT", 30)),
-    }
+    NETWORK = NETWORKS["katana"]
```

Comment:
Just saw @Eikix comment but this is actually not the right strategy as this kills the following existing feature : set up RPC_URL only and it works.

So you want to put katana if neither STARKNET_NETWORK nor RPC_URL are set